pkg/asset/machines/machineconfig/workload-partitioning.go

```diff
@@ -0,0 +1,127 @@
+package machineconfig
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	ini "gopkg.in/ini.v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/installer/pkg/asset/ignition"
+	"github.com/pkg/errors"
+)
+
+// CrioWorkloadDropinContents generates the content expected by Cri-O for workload partitioning
+//
+// Example output:
+// [crio.runtime.workloads.management]
+// label = "management.workload.openshift.io/cores"
+// annotation_prefix = "io.openshift.workload.management"
+// resources = { "cpu" = "", "cpuset" = "0-1", }
+func CrioWorkloadDropinContents(name, cpuset string) (string, error) {
+	/*
+		[crio.runtime.workloads.management]
+		label = "management.workload.openshift.io/cores"
+		annotation_prefix = "io.openshift.workload.management"
+		resources = { "cpu" = "", "cpuset" = "0-1", }
+	*/
+	crioIni := ini.Empty()
+	section := crioIni.Section(fmt.Sprintf("crio.runtime.workloads.%s", name))
+	err := section.ReflectFrom(&crioWorkloadCfg{
+		Label:            fmt.Sprintf(`"%s.workload.openshift.io/cores"`, name),
+		AnnotationPrefix: fmt.Sprintf(`"io.openshift.workload.%s"`, name),
+		Resources:        fmt.Sprintf(`{ "cpu" = "", "cpuset" = "%s", }`, cpuset),
+	})
+	if err != nil {
+		return "", errors.Wrap(err, "Could not reflect structure to INI")
```

Comment:
Errors should start with lowercase.
```suggestion
		return "", errors.Wrap(err, "could not reflect structure to INI")
```